Games/baj.2m.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# <bitbar.title>BAJ</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Justin Honold</bitbar.author>
+# <bitbar.author.github>astrostl</bitbar.author.github>
+# <bitbar.desc>Checks for active turns at http://boiteajeux.net every two minutes</bitbar.desc>
+# <bitbar.image></bitbar.image>
```

Comment:
Hi @astrostl, one last thing. Could you please add an image to this, that'd be awesome!